core/src/main/resources/xml/xsl/web/adapterSite.xsl

```diff
@@ -467,6 +467,7 @@
 			<xsl:when test="@type='showsummary'"		>images/ibisstore.gif</xsl:when>
 			<xsl:when test="@type='theme'"				>images/new-gui.png</xsl:when>
 
+
```

Comment:
Please try to refrain from committing empty lines

Reply:
I'll hold this in mind when committing, my bad.